src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -201,6 +203,13 @@
       InputListLocation="$(TestDependencyListFilePath)"
       OutputListLocation="$(OutDir)assemblylist.txt"
      />
+      
+    <!-- Additional execution commands included for specific scenarios -->
+    <!-- Testing with AppX -->
+    <ItemGroup>
+      <AppXCopyCommands Condition="'$(TestAppX)' == 'true'" Include="xcopy &quot;%PACKAGE_DIR%$([System.String]::Copy('%(_TestCopyLocalIncludeByForce.SourcePath)').Replace('$(PackagesDir)',''))&quot; &quot;%EXECUTION_DIR%$([System.String]::Copy('%(_TestCopyLocalIncludeByForce.SourcePath)').Replace('$(PackagesDir)microsoft.xunit.runner.uwp\$(AppXRunnerVersion)\',''))*&quot; /y"></AppXCopyCommands>
```

Comment:
Why do you need to explicitly make a copy command here? I would expect you just add it the list of files that we already put in the copy list.